charts/fission-all/templates/webhook-server/clusterrole.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-webhook
+rules:
```

Comment:
Please remove unwanted permissions from this role.

Reply:
@sanketsudake all the validation in web-hooks are spec checks. And none of these permissions are required. Should I completely remove this cluster-role?